tests/zenko_tests/node_tests/iam_policies/cloudserver/AssumeRoleWithWebIdentity.js

```diff
@@ -21,117 +22,149 @@ const externalSecretKey = 'iTuJdlidzrLipymvAGrLP66Yxghl4NQxLZR3cLlu';
 
 const duration = '1000';
 
-const bucket1 = 'bucket1';
-
 const storageManagerName = 'storage_manager';
 const storageAccountOwnerName = 'storage_account_owner';
 const dataConsumerName = 'data_consumer';
 const storageManagerRole = 'storage-manager-role';
 const storageAccountOwnerRole = 'storage-account-owner-role';
 const dataConsumerRole = 'data-consumer-role';
 
-describe('iam policies - cloudserver - AssumeRoleWithWebIdentity - Metadata', () => {
+const testAPIs = [
+    {
+        API: 'MetadataSearch',
+        checkResponse: metadataSearchResponseCode,
+    },
+    {
+        API: 'RestoreObject',
+        checkResponse: restoreObjectResponseCode,
+    },
+];
```

Comment:
A comment would be helpful in explaining, that we can add new APIs here. (Intent)